Gemfile

```diff
@@ -114,3 +116,10 @@ gem 'devise', '~> 4.9', '>= 4.9.2'
 gem 'pry-rails', :group => :development
 
 gem "jquery-rails"
+
+gem 'dotenv-rails'
+gem 'omniauth'
```

Comment:
This gem has dependency relationship with omniauth-facebook, omniauth-github, omniauth-google-oauth2.